tests/test_environs.py

```diff
@@ -214,6 +214,16 @@ def test_reading_proxied_variable_in_prefix_scope(self, set_env, env):
                 assert env.str("LOGIN") == "szabolcs"
                 assert env.str("PASSWORD") == "nested-secret"
 
+class TestSpecificFileReading:
+    def test_read_env(self, env):
+        assert env("STRING", "default") == "default"  # sanity check
+        env.read_env('tests/.custom.env')
+        assert env("CUSTOM_STRING") == "foo"
+
+    def test_read_env_non_recurse(self, env):
+        assert env("STRING", "default") == "default"  # sanity check
```

Comment:
Not sure these assertions are necessary. What is the intent here?

Reply:
You're right. The sanity check was wrong.